### PR TITLE
fix: change email verification success msg

### PR DESCRIPTION
**Pull request by bsanket16**
Fixes #
Email verification message on successful verification of users #4989 

Showing Email Verification message after checking authentication

case1 : Authenticated
Message: Your email has been successfully verified!

case2: Not Authenticated
Message: Your email has been successfully verified! Please login to continue.

#### Short description of what this resolves:
Altered success message on email verification.

### CODE REVIEW

**Review by Raunaksingh100 — COMMENTED** [review]
@bsanket16  provide the screenshot  in the pr description what you fixed

**Comment by iamareebjamal** [comment]
You haven't marked which issue it fixes

**Comment by Raunaksingh100** [comment]
@iamareebjamal  can you review 
https://github.com/fossasia/open-event-frontend/pull/4976

File: app/templates/verify.hbs [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
+  {{#if this.success}}
+    <div class="ui {{if this.isLoading 'loading'}} icon info message eight wide column center aligned">
+      <p>Your email has been verified successfully! Please login to continue</p>
+    </div>
+  {{else}}
+    <div class="ui {{if this.isLoading 'loading'}} icon error message eight wide column center aligned">
+      <p>Error: {{t this.error}}</p>
+    </div>
+  {{/if}}
+{{/unless}}
```
Comment (snitin315) [inline app/templates/verify.hbs:23]: I can see duplication here, please remove it and merge into one `if-else` block.

Comment (bsanket16) [reply app/templates/verify.hbs]: Done. Please have a look at it now.

**Pull request closed without merging**